### PR TITLE
finally

### PR DESCRIPTION
**Pull request by despair86**
got it to build manually now

### CODE REVIEW

File: src/process/WindowsLokinetProcessManager.cpp [hunk 1]
```diff
@@ -73,7 +73,7 @@ bool WindowsLokinetProcessManager::doGetProcessPid(int& pid)
     {
         while (Process32Next(snapshot, &entry) == TRUE)
         {
-            if (wcscmp(entry.szExeFile, L"lokinet.exe") == 0)
+            if (stricmp(entry.szExeFile, "lokinet.exe") == 0)
```
Comment (despair86) [inline src/process/WindowsLokinetProcessManager.cpp:76]: only msvc bothered to set `-municode`/`-DUNICODE`

Comment (despair86) [reply src/process/WindowsLokinetProcessManager.cpp:76]: mingw by default uses the ANSI function signatures for better UTF-8 support

File: .drone.jsonnet [hunk 2]
```diff
@@ -173,11 +184,11 @@ local deb_builder(image, distro, distro_branch, arch='amd64', imaginary_repo=tru
...
-                    '../contrib/ci/drone-check-static-libs.sh',
-                    '../contrib/ci/drone-static-upload.sh'
-                ]),
+    //mac_builder('macOS (Release)'),
+    //mac_builder('macOS (Debug)', build_type='Debug'),
+    //mac_builder('macOS (Static)',
+//                extra_cmds=[
+//                    '../contrib/ci/drone-check-static-libs.sh',
+//                    '../contrib/ci/drone-static-upload.sh'
+//                ]),
```
Comment (despair86) [inline .drone.jsonnet:178]: if you want to restore these some other time lmk

**Comment by despair86** [comment]
@majestrate did you ever fix the cmake rules in this repo

**Comment by majestrate** [comment]
what was broken about it?

**Comment by despair86** [comment]
same problem you fixed earlier this week with broken ci

**Comment by majestrate** [comment]
the libsodium thing?

**Comment by despair86** [comment]
yes

On September 25, 2020 4:12:47 PM CDT, Jeff <notifications@github.com> wrote:
>the libsodium thing?>
>>
>On Fri, Sep 25, 2020 at 14:04, Rick V <notifications@github.com>
>wrote:>
>> same problem you fixed earlier this week with broken ci>
>> >
>> —>
>> You are receiving this because you were mentioned.>
>> Reply to this email directly, view it on GitHub, or unsubscribe.>
>> >
>>
>>
>>
>>
>-- >
>You are receiving this because you authored the thread.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/loki-project/loki-network-control-panel/pull/44#issuecomment-699157728

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by majestrate** [comment]
just merged the libsodium stuff into master, build flag for it is `-DBUILD_STATIC_DEPS=ON`

**Review by majestrate — APPROVED** [review]

**Pull request merged**